pkg/datasources/streamlits_acceptance_test.go

```diff
@@ -0,0 +1,238 @@
+package datasources_test
+
+import (
+	"context"
+	"fmt"
+	"maps"
+	"regexp"
+	"testing"
+
+	acc "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/internal/snowflakeroles"
+	"github.com/stretchr/testify/require"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/helpers/random"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/provider/resources"
+	"github.com/hashicorp/terraform-plugin-testing/config"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/tfversion"
+)
+
+func TestAcc_Streamlits(t *testing.T) {
+	databaseId := acc.TestClient().Ids.DatabaseId()
+	schemaId := acc.TestClient().Ids.SchemaId()
+	id := acc.TestClient().Ids.RandomSchemaObjectIdentifier()
+	acc.TestAccPreCheck(t)
+	stage, stageCleanup := acc.TestClient().Stage.CreateStageInSchema(t, schemaId)
+	t.Cleanup(stageCleanup)
+	// use schema is needed because otherwise reference to external access integration fails.
+	err := acc.Client(t).Sessions.UseSchema(context.Background(), schemaId)
```

Comment:
it should not be needed because we create it, so it is used by default.

Reply:
Right, removed.